### DISCUSSION TITLE
Rouvy compatibility

### DISCUSSION THREAD

**Discussion by DStaz** [cagnulein/qdomyos-zwift#2826] [General]
Is Rouvy really compatible with this app. I don't have any resistance coming from Rouvy. Flat road or ascension are the same 😢

**Comment by cagnulein** [comment]
yes of course! did you pair qz as controllable? are you on android or ios? create a debug log so i can check your settings with the problem https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

**Comment by Vendo233** [comment]
I tested Rouvy with QZnand it worked flawlessly

**Reply by cagnulein** [reply]
Why didn't you simply write here for help instead of spending time alone, we are here to help! @indoorjohnny 
For reset the app simply create a new profile and it will ask to reset everything then restart the app.

On which device are you running rouvy?

**Comment by indoorjohnny** [comment]


**Reply by cagnulein** [reply]
no problem. Let me help you

**Comment by indoorjohnny** [comment]
How do I reset the app so I can start over like a new person.

**Reply by cagnulein** [reply]
go into the left side bar, profile, create a new profile and restart the app so all the settings will be set to the default ones.
